EventListener/TranslationPostPushEvent.php

```diff
@@ -25,16 +25,13 @@ public function __construct(array $uploadedFiles = [])
         $this->uploadedFiles = $uploadedFiles;
     }
 
-    /**
-     * @return string
-     */
-    public static function getEventName()
+    public static function getEventName(): string
     {
         return self::EVENT_NAME;
     }
 
     /**
-     * @return array|\OpenClassrooms\Bundle\OneSkyBundle\Model\UploadFile[]
+     * @return array|UploadFile[]
```

Comment:
Pretty sure the original typehint was wrong here, and we could delete the `array|` part from it (so it matches the type of `$this->uploadedFiles`)